Dockerfile

```diff
@@ -7,37 +7,9 @@ MAINTAINER Gammapy developers <gammapy@googlegroups.com>
 # compilers
 RUN apt-get install -y build-essential
 
-# install dependencies
-RUN conda config --add channels conda-forge
-RUN conda config --add channels astropy
-RUN conda config --add channels sherpa
-
-RUN conda install -q -y python==3.6
-RUN conda install -q -y cython>=0.27
-RUN conda install -q -y numpy>=1.13
-RUN conda install -q -y astropy
-RUN conda install -q -y uncertainties
-RUN conda install -q -y scipy>=0.19.1
-RUN conda install -q -y scikit-image
-RUN conda install -q -y matplotlib>=2.0
-RUN conda install -q -y astropy>=2.0
-RUN conda install -q -y regions>=0.2
-RUN conda install -q -y reproject
-RUN conda install -q -y iminuit
-RUN conda install -q -y healpy
-RUN conda install -q -y photutils
-RUN conda install -q -y aplpy
-RUN conda install -q -y pyyaml
-RUN conda install -q -y sherpa>=4.9.1
-RUN conda install -q -y click
-
-# install good version of Jupyter notebook
-RUN pip install --no-cache-dir notebook==5.*
-
-# install last version of gammapy
-RUN /bin/bash -c "git clone https://github.com/gammapy/gammapy.git && \
-                  cd gammapy && \
-                  python setup.py install"
+# install dependencies, including the dev version of Gammapy
+RUN conda env create -f environment.yml
```

Comment:
The Docker build process in Binder throws the following error:
`The command '/bin/sh -c conda env create -f environment.yml' returned a non-zero code: 1`

You can test yourself the compilation wit the following steps:
1. https://mybinder.org
2. Provide URL of your GitHub repo: `https://github.com/cdeil/gammapy-extra.git`  
3. Provide also the Git branch: `conda-env` 
4. Click on launch bouton
5. Follow the logs clicking on 'Build logs'